src/audio/tdfb/tdfb.c

```diff
@@ -380,6 +690,27 @@ static int tdfb_cmd(struct comp_dev *dev, int cmd, void *data,
 	return ret;
 }
 
+/* TODO: Replace algorithm mockup. This only enables testing with controls. */
+static void update_direction_of_arrival(struct tdfb_comp_data *cd)
```

Comment:
is this for upstreaming ??

Reply:
It is to enable testing of @ujfalusi 's patch. ETA for real angle values calculation is August. If preferred I can remove the fake angles lottery and people who need to test can patch this back. What's your proposal?

Reply:
remove the fake

Reply:
Done, I'll get something done to help test this feature.